drivers/platform/linux/linux_gpio.c

```diff
@@ -73,34 +89,59 @@ int32_t gpio_get(struct gpio_desc **desc,
 	if (!descriptor)
 		return FAILURE;
 
+	linux_desc = (struct linux_gpio_desc*) malloc(sizeof(struct linux_gpio_desc));
```

Comment:
```suggestion
	linux_desc = (struct linux_gpio_desc*)calloc(1, sizeof(*linux_desc));
```
To use *linux_desc is how linux coding style recommands it
Calloc is prefered because it initializes the memory with 0 and some errors in code can be found faster